hack/scripts/check_artifacts_existence.sh

```diff
@@ -21,8 +21,7 @@ IMAGE_NAMES=(
 
 for image in "${IMAGE_NAMES[@]}"; do
   if [ $(skopeo list-tags ${PROTOCOL}europe-docker.pkg.dev/kyma-project/prod/${image} | jq '.Tags|any(. == env.IMAGE_TAG)') == "true" ]; then
-    echo "::warning ::${image} binary image for tag ${IMAGE_TAG} already exists"
-    exit 1
+    echo "::WARNING ::${image} binary image for tag ${IMAGE_TAG} already exists"
```

Comment:
Not exiting with 1 so that we can run again the release pipeline for same version in case the pipeline failed. Same policy as in KEB [here](https://github.com/kyma-project/kyma-environment-broker/blob/26b1ce02c2de48ba2712b010951dbc233644a18c/scripts/check_artifacts_existence.sh#L29).